src/lvl_script.c

```diff
@@ -6613,6 +6691,36 @@ void script_process_value(unsigned long var_index, unsigned long plr_range_id, l
           set_variable(i, val4, val2, (rand() % val3) + 1);
       }
       break;
+  case Cmd_SET_FLAG_COMPUTED:
+      {
+        long src_plr_range = (val2 >> 24) & 255;
+        long operation = (val2 >> 16) & 255;
+        unsigned char flag_type = (val2 >> 8) & 255;
+        unsigned char src_flag_type = val2 & 255;
+        int src_plr_start, src_plr_end;
+        if (get_players_range(src_plr_range, &src_plr_start, &src_plr_end) < 0)
+        {
+            WARNLOG("Invalid player range %d in VALUE command %d.",(int)src_plr_range,(int)var_index);
+            return;
+        }
+        long sum = 0;
+        for (i=src_plr_start; i < src_plr_end; i++)
+        {
+            sum += get_condition_value(i, src_flag_type, val4);
+        }
+        for (i=plr_start; i < plr_end; i++)
+        {
+            long current_flag_val = get_condition_value(i, flag_type, val3);
+            long computed = sum;
+            if (operation == SOpr_INCREASE) computed = current_flag_val + sum;
```

Comment:
Why not SOpr_SET too?

Reply:
Well, I don't know why, but since I suspected it might work anyway, I tested it and it does work.